frame/pallet-xcm/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+name = "pallet-xcm"
+version = "0.9.19"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
```

Comment:
We use `2.1.0` in other pallets. Better to be consistent.

Reply:
Fixed, thank you!